test/jpypetest/test_jchar.py

```diff
@@ -56,9 +54,10 @@ def testJPJavaFrameCharArray(self):
         _jpype.fault("JPJavaFrame::ReleaseCharArrayElements")
         with self.assertRaisesRegex(SystemError, "fault"):
             ja[0:3] = bytes([1, 2, 3])
-        _jpype.fault("JPJavaFrame::ReleaseCharArrayElements")
-        with self.assertRaisesRegex(SystemError, "fault"):
-            jpype.JObject(ja[::2], jpype.JObject)
+        # Not sure why this one changed.
```

Comment:
So it does not trigger the releaseCharArrayElements any more? Which wrapper is used instead?

Reply:
Hmmm.  Good question.   I will track this one down and make sure  there is not a new leak here.    Nothing I was touching in this PR should have touched the array paths.  But a closer look is warrented.

Reply:
This was a transient problem (happens when I jump branches, compile, then jump back.)  Re-enabled the test and all is good.